Build Error on API.Tests

##[error]Covid19RadarAPI.Tests\UserApiTest.cs(17,31): Error CS7036: There is no argument given that corresponds to the required formal parameter 'logger' of 'UserApi.UserApi(ICosmos, INotificationService, IValidationUserService, ILogger<UserApi>)'

--- a/Covid19RadarAPI.Tests/Mock/ValidationUserServiceMock.cs
+++ b/Covid19RadarAPI.Tests/Mock/ValidationUserServiceMock.cs
@@ -1,0 +1,18 @@
+﻿using Covid19Radar.Models;
+using Covid19Radar.Services;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Covid19Radar.Tests.Mock
+{
+    public class ValidationUserServiceMock : IValidationUserService
+    {
+        public Task<IValidationUserService.ValidateResult> ValidateAsync(HttpRequest req, IUser user)
+        {
+            return Task.FromResult(new IValidationUserService.ValidateResult() { IsValid = true, User = new UserModel(), ErrorActionResult = null });
+        }
+    }
+}

--- a/Covid19RadarAPI.Tests/UserApiTest.cs
+++ b/Covid19RadarAPI.Tests/UserApiTest.cs
@@ -13,8 +13,9 @@ namespace Covid19Radar.Tests
             // preparation
             var cosmos = new Mock.CosmosMock();
             var notification = new Mock.NotificationServiceMock();
+            var validation = new Mock.ValidationUserServiceMock();
             var logger = new Mock.LoggerMock<UserApi>();
-            var userApi = new UserApi(cosmos, notification, logger);
+            var userApi = new UserApi(cosmos, notification, validation, logger);
         }
 
         [TestMethod]
@@ -23,8 +24,9 @@ namespace Covid19Radar.Tests
             // preparation
             var cosmos = new Mock.CosmosMock();
             var notification = new Mock.NotificationServiceMock();
+            var validation = new Mock.ValidationUserServiceMock();
             var logger = new Mock.LoggerMock<UserApi>();
-            var userApi = new UserApi(cosmos, notification, logger);
+            var userApi = new UserApi(cosmos, notification, validation, logger);
             var context = new Mock.HttpContextMock();
             // action
             userApi.Run(context.Request);

